src/fmu/dataio/dataio.py

```diff
@@ -698,7 +732,11 @@ def generate_metadata(self, obj: Any, compute_md5: bool = True, **kwargs) -> dic
 
         self._update_check_settings(kwargs)
         self._update_globalconfig_from_settings()
-        _check_global_config(self.config)
+
+        self._metadata_is_valid = _check_global_config(
```

Comment:
Something about the variable/function names that does not resonate with me her, but perhaps there is a logic I have not seen. The `self._metadata_is_valid` is really referring to the config, not the metadata? (The config _contains_ some metadata, so on some level it makes sense, but I think this may be confusing since _actual_ metadata is being produced - and that can be invalid (according to the schema)).

Reply:
Good point, changed to `_config_is_valid`